assets/css/base/_base.scss

```diff
@@ -1357,6 +1357,7 @@ button.menu-toggle {
 	.page-numbers {
 		list-style: none;
 		margin: 0;
+		vertical-align: middle;
```

Comment:
After adding `overflow: hidden;` to `.next` and `.prev` links, they were vertically misaligned. Applying `vertical-align: middle` to all items fixes that.